### DISCUSSION TITLE
Modification of external projects

### DISCUSSION THREAD

**Discussion by BlackxSnow** [premake/premake-core#1846] [Q&A] [answered]
Is it possible to modify an external project's configuration? I'd like to add a post-build comment to GENie generated project file, but simply calling the function after the project declaration doesn't seem to do anything (nor does it throw errors).

**Comment by nickclark2016** [comment, marked as answer]
This is not something that we support. When we add an external project, it's consumed as-is. We don't read in the project file contents to support manipulation.

**Reply by BlackxSnow** [reply]
Oh, that's a shame. I'd love to see that as a feature at some point in the future (premake 6 anyone?)

Thanks for the answer!

**Reply by nickclark2016** [reply]
Just to follow up on this, the issue I see with this is that we'd need to write parsers for each project type that we support.  That's currently out of scope.  That said, it could be an interesting 3rd party module.
